worker/config/p2p/testnet_config.json

```diff
@@ -1,28 +1,29 @@
-{"CONTRACT_DISCOVERY_ADDRESS": "http://contract:8081",
- "ETH_NODE_ADDRESS": "",
- "ENIGMA_CONTRACT_FILE_NAME": "Enigma.json",
- "CORE_ADDRESS": "localhost",
- "CORE_PORT": "5552",
- "CONTRACTS_FOLDER": "/root/p2p/contracts/",
- "KEY_MANAGEMENT_ADDRESS": "http://km-service:3040",
- "KEY_MANAGEMENT_DISCOVERY": "http://km-service:8081",
- "FAUCET_URL": "",
- "MINIMUM_ETHER_BALANCE": "0.1",
- "MINIMUM_ENG_BALANCE": "100000000000",
- "DEPOSIT_AMOUNT": 100000000000,
- "ETH_KEY_PATH": "/root/.enigma/",
- "STAKE_KEY_PATH": "/root/.enigma/staking/",
- "BOOTSTRAP_ID": "B1",
- "BOOTSTRAP_PORT": "10300",
- "BOOTSTRAP_PATH": "/root/p2p/scripts/bootstrap/",
- "LIBP2P_KEY_PATH": "/root/.enigma/libp2p/",
- "BALANCE_WAIT_TIME": 1800,
- "MIN_CONFIRMATIONS": 12,
- "KEY_MANAGEMENT_TIMEOUT": 120,
- "CONTRACT_TIMEOUT": 3600,
- "HEALTH_CHECK_PORT": 12345,
- "STAKE_KEY_NAME": "staking_address",
- "STATUS_FILENAME": "status",
- "ETHEREUM_ADDR_FILENAME": "eth_address.txt",
- "MGMT_URL": "http://localhost:23456/mgmt/"
-}
+{
+    "CONTRACT_DISCOVERY_ADDRESS": "http://contract:8081",
+    "ETH_NODE_ADDRESS": "http://parity.enigma.co:8545",
+    "ENIGMA_CONTRACT_FILE_NAME": "Enigma.json",
+    "CORE_ADDRESS": "localhost",
+    "CORE_PORT": "5552",
+    "CONTRACTS_FOLDER": "/root/p2p/contracts/",
+    "KEY_MANAGEMENT_ADDRESS": "http://km-testnet.enigma.co:3040",
+    "KEY_MANAGEMENT_DISCOVERY": "http://km-service:8081",
+    "FAUCET_URL": "",
+    "MINIMUM_ETHER_BALANCE": "0.1",
+    "MINIMUM_ENG_BALANCE": "100000000000",
+    "DEPOSIT_AMOUNT": 100000000000,
+    "ETH_KEY_PATH": "/root/.enigma/",
+    "STAKE_KEY_PATH": "/root/.enigma/staking/",
+    "BOOTSTRAP_ID": "B1",
```

Comment:
Don't need any BOOTSTRAP_ for testnet

Reply:
only BOOTSTRAP_ID?

Reply:
Yes but that one is set from the kubernetes deployment